### PR TITLE
update deps

### PR DESCRIPTION
**Pull request by tylerl0706**
Updated all of our npm dependencies

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5,15 +5,15 @@
   "requires": true,
   "dependencies": {
     "@types/mocha": {
-      "version": "5.2.5",
-      "resolved": "https://registry.npmjs.org/@types/mocha/-/mocha-5.2.5.tgz",
-      "integrity": "sha512-lAVp+Kj54ui/vLUFxsJTMtWvZraZxum3w3Nwkble2dNuV5VnPA+Mi2oGX9XYJAaIvZi3tn3cbjS/qcJXRb6Bww==",
+      "version": "2.2.48",
```
Comment (rjmholt) [inline package-lock.json]: What is going on here?

Comment (rjmholt) [reply package-lock.json]: (I know it's a dev dependency, but still seems like a weird version "increment")

Comment (tylerl0706) [reply package-lock.json]: The package-lock.json works in mysterious ways...

Comment (rkeithhill) [reply package-lock.json]: The inner machinations of npm are an enigma...

**Pull request merged**